qemu/tests/enforce_quit.py

```diff
@@ -21,6 +21,7 @@ def run(test, params, env):
     host_cpumodel = utils_misc.get_host_cpu_models()
     host_flags = utils_misc.get_cpu_flags()
     extra_flags = params.get("cpu_model_flags", " ")
+    no_check_flags = params.get("no_check_flags", " ").split()
```

Comment:
it's better to give some comment to no_check_flags
and in my opinion, we can filter no_check_flags here, 
also filter "-flags" instead of filter "+svm" directly below